vercel.json

```diff
@@ -823,7 +823,8 @@
         { "source": "/products/product-analytics", "destination": "/product-analytics" },
         { "source": "/session-recording", "destination": "/session-replay" },
         { "source": "/manual/recordings", "destination": "/docs/session-replay/manual" },
-        { "source": "/docs/session-recording", "destination": "/docs/session-replay" }
+        { "source": "/docs/session-recording", "destination": "/docs/session-replay" },
+        { "source": "/handbook/small-teams/experimentation", "destination": "/handbook/small-teams/feature-success" }
```

Comment:
Are these redirects transitive?

Ex: there's a previous redirect from 

```
{ "source": "/handbook/engineering/app-west", "destination": "/handbook/small-teams/experimentation" },
```

and now this one, so will `engineering/app-west` redirect to the right place?

Reply:
Yep tried it on the deploy, works!